docs/security/seccomp.md

```diff
@@ -16,10 +16,16 @@ restrict the actions available within the container. The `seccomp()` system
 call operates on the seccomp state of the calling process. You can use this
 feature to restrict your application's access.
 
-This feature is available only if the kernel is configured with `CONFIG_SECCOMP`
-enabled.
+This feature is available only if Docker has been built with seccomp and the
+kernel is configured with `CONFIG_SECCOMP` enabled. To check if your kernel
+supports seccomp:
 
-> **Note**: Seccomp profiles require seccomp 2.2.1 and are only
+```
```

Comment:
fixed markdown hint.

wrt check-config.sh - I think here for seccomp it's easy to just say check kernel config instead of running the script (and get lost in the output)